src/EventStore.Core/Services/SubscriptionMetric.cs

```diff
@@ -0,0 +1,74 @@
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using EventStore.Core.Messages;
+
+namespace EventStore.Core.Services;
+
+public class SubscriptionMetric {
+	private readonly ConcurrentDictionary<Guid, MonitoringMessage.SubscriptionStats> _subscriptionStats;
+
+	public SubscriptionMetric(Meter meter, string name) {
+		_subscriptionStats = new();
+
+		meter.CreateObservableUpDownCounter($"{name}-completed", Completed);
```

Comment:
it might be better to leave this as a calculation to be done in the dashboard, since its just derived from the other metrics

i filed a task to update the grafana dashboard actually, it might be worth doing that to sanity check that it's reasonble to produce the graphs we want from these metrics